lekce/3/javascript.md

```diff
@@ -22,8 +22,6 @@ HTML, CSS a JavaScript se interpretují v prohlížeči u uživatele, takže se
 
 ----
 
-Stránka může…
```

Comment:
To jo, ale tady to nesedí hned s druhou odrážkou, tak jsem zvolil jednoduchý řešení